### ISSUE TITLE
Kanji dictionary integration to translator

### ISSUE THREAD

**Issue by bayazidbh** [OneMoreGres/ScreenTranslator#53]
**Is your feature request related to a problem? Please describe.**
The online translators are as usual very spotty - it gets the general idea of what the sentence is supposed to mean decently enough, but if one only have problems with a few kanji, then it's better to have a way to understand that kanji and then piece together the sentence yourself.

**Describe the solution you'd like**
Some sort of kanji dictionary integration to the translator window.

**Describe alternatives you've considered**
As the kanji translator seems to be a browser of sort, then perhaps integration with [Yomi-chan ](https://foosoft.net/projects/yomichan/)or [Rikaichamp](https://github.com/birtles/rikaichamp)?

Alternatively, a custom translator that pulls from an offline dictionary (integrated as a translator) of a sort?

**Comment by OneMoreGres** [comment]
Hi!
I have per word translators/dictionaries in wish-list, but I don't have any concrete plans how or when to do it.
So far it's `someday/maybe`.

As a workaround, you can select and translate a single word (kanji).

You can also open the translator's window via tray icon's context menu. Some translation services (like google) show word's translation when you select it.

**Comment by Enferlain** [comment]
Yeah for this currently Google is probably the best option. DeepL is amazing if you just want to read and the quality is actually decent, but it doesn't show the latin letters so it's not good for learning kanji. Google option is good for that.
